modules/issue_tracker/jsx/issueIndex.js

```diff
@@ -474,24 +474,82 @@ class CollapsibleComment extends React.Component {
       "Show Comment History" :
       "Hide Comment History");
 
+    const hrStyle = {
+      marginTop: "8px",
+      marginBottom: "8px"
+    };
+    const colStyle = {
+      paddingRight: "8px"
+    };
     const commentHistory = this.props.commentHistory;
+    let curDateAddedF = null;
+    let curDateAdded = null;
+    let curAddedBy = null;
+    let curState = null;
+    let fieldUsed = null;
+    let fieldChangeArr = null;
+    commentHistory.push({
```

Comment:
@jacobpenny Will you accept a hacky `const commentHistory = this.props.commentHistory.clone();` ?